src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -152,9 +152,10 @@ object DockerPlugin extends AutoPlugin {
     ).toOption
       .map(_.trim)
       .flatMap(DockerApiVersion.parse),
+    dockerBuildInit := false,
```

Comment:
Maybe we can use the `dockerVersion` or `dockerApiVersion` to set this true when possible?

Reply:
Possible, however, `dockerVersion` and `dockerApiVersion` are task keys. `dockerBuildOptions` and `dockerBuildCommand` are setting keys, and so can't depend on `dockerVersion` or `dockerApiVersion`. We'd have to depend on this in `publishLocal`, which I think would be bad because it would mean `dockerBuildCommand` would no longer be the source of truth as to what was executed.